packages/components/forms/src/Label/README.mdx

```diff
@@ -66,3 +70,7 @@ Use this guidance to determine the correct required properties to use based on t
 ## Accessibility
 
 Missing
```

Comment:
should we address this, or README get refreshed as part of a different effort?

Reply:
I would push that to later since this component is blocking the other input PRs :/

Reply:
Since there is a lot of stuff we want to synchronize later, I'm wondering if we already have some kind of list of EPIC to do all those things (file names & locations, documentation, prop names, etc.)